index.html

```diff
@@ -0,0 +1,171 @@
+                 и готово.
+            </p>
+            <span class="advantages__backgr-number">2</span>
+        </div>
+        <div class="advantages__card">
+            <p class="advantages__description">Замените один-два приема обычной еды на наше фунцкциональное питание.
+            </p>
+            <span class="advantages__backgr-number">3</span>
+        </div>
+        <div class="advantages__card">
+            <p class="advantages__description">Уже через месяц наслаждайтесь изменениями к лучшему 
+                вашего питомца!
+            </p>
+            <span class="advantages__backgr-number">4</span>
+        </div>
+      </section>
+
+      <section class="example">
+          <h2 class="example__title title">Живой пример</h2>
+          <div class="example__wrap container">
+              <div class="example__text-wrap">
+                  <p class="example__description">Борис сбросил 5 кг за 2 месяца, просто заменив свой обычный
+                       корм на Cat Energy Slim. Отличный результат без изнуряющих тренировок! При этом он не 
+                       менял своих привычек и по-прежнему спит по 16 часов в день.</p>
+                       <div class="example__means">
+                           <div class="example__mean">
+                               <p class="example__main-text">5 кг</p>
+                               <p class="example__secondary-text">снижение веса</p>
+                           </div>
+                           <div class="example__mean">
+                            <p class="example__main-text">60 дней</p>
+                            <p class="example__secondary-text">затрачено времени</p>
+                            </div>
+                       </div>
+                       <div class="example__total">
+                           <p class="example__expenses">Затраты на питание: </p>
+                           <p class="example__sum">15 000 руб.</p>
+                       </div>
+              </div>
+              <div class="example__before-after">
+                  <img
+                  class="example__weight-cat"
+                  src="./img/before-desktop.svg"
+                  width="365"
+                  height="536"
+                  alt="before_cat"
+                />
+                <img
+                class="example__weight-cat"
+                src="./img/after-desktop.svg"
+                width="325"
+                height="654"
+                alt="after_cat"
+              />
+              </div>
+          </div>
+      </section>
+
+      <section class="map">
+        <div class="map__content">
+          <p class="map__cooperation">Приглашаем 
+            к сотрудничеству 
+            дилеров!</p>
+            <div class="map__text-wrap">
+              <p class="map__adress">
+                ул. Большая Конюшенная, д. 19/8
+              </p>
+              <p class="map__country">
```

Comment:
If you look at the design, these lines of text look very similar. The main rule is to set same CSS classes to elements which look the same. If you need to distinguish them on the styling phase, you can add modifiers to them (or one of them).

Reply:
++